src/vic3/tables/triggers.rs

```diff
@@ -805,3 +810,24 @@ const TRIGGER: &[(Scopes, &str, Trigger)] = &[
     (Scopes::None, "weighted_calc_true_if", Special),
     (Scopes::None, "year", CompareValue),
 ];
+
+pub fn scope_trigger_special_value(name: &Token) -> Option<(Scopes, Trigger)> {
+    let name_lc = name.as_str().to_lowercase();
+    TRIGGER_SPECIAL_VALUE_MAP.get(&*name_lc).map(Clone::clone)
+}
+
+static TRIGGER_SPECIAL_VALUE_MAP: Lazy<FnvHashMap<&'static str, (Scopes, Trigger)>> =
+    Lazy::new(|| {
+        let mut hash = FnvHashMap::default();
+        for &(from, s, trigger) in TRIGGER_SPECIAL_VALUE {
+            hash.insert(s, (from, trigger));
+        }
+        hash
+    });
+
+/// LAST UPDATED VIC3 VERSION N/A
+/// See `triggers.log` from the game data dumps
+/// `(inscopes, trigger name, argtype)`
+/// Currently only works with single argument triggers
+// TODO Verify triggers
+const TRIGGER_SPECIAL_VALUE: &[(Scopes, &str, Trigger)] = &[];
```

Comment:
Hmm this is a regression since you did remove them from handle_argument. Can you put those in the table?

Reply:
Some of the argument validation requires updating vic3 as a whole to 1.5, i.e. adding new items and scopes. I will leave out those as `UncheckedValue` for now and leave a TODO for when it is updated.

Reply:
In addition, I think entries from the `event_targets.log` which have output scope of value (such as `num_mutual_trade_route_levels_with_country` should *not* be mixed with the entries from `triggers`, but within their own table in `scopes.rs` instead.